syncer.py

```diff
@@ -336,16 +377,45 @@ async def start(self):
                 icon="fas fa-bed",
             )
 
+            body_data = {
+                "weight": await self.prepare(
+                    "weight",
+                    "fitbit body weight",
+                    "Weight",
+                    "",
+                    {"type": "number"},
+                    icon="fas fa-weight-scale",
```

Comment:
The icons for BMI and weight are not showing up for me in my test database. The icons do seem to exist, so it looks like it might be an issue on my end.

Reply:
No, you're right, I missed it/didn't test it properly. `fa-weight-scale` seems to be a version 6 icon, but heedy uses v5 (free). I changed the icons to v5 free ones.